packages/xstate-test/src/TestModel.ts

```diff
@@ -176,10 +171,8 @@ export class TestModel<TState, TEvent extends EventObject> {
     return Object.values(adj).map((x) => x.state);
   }
 
-  public async testPlans(
-    plans: Array<StatePlan<TState, TEvent>>,
-    options?: Partial<TestModelOptions<TState, TEvent>>
-  ) {
+  public async testPlans(options?: TestPlansOptions<TState, TEvent>) {
```

Comment:
`testPlans` now takes plans from an options object instead of its first argument.

Reply:
It means that the simplest API is:

```ts
const machine = createTestMachine();

const model = createTestModel(machine);

it('should work', model.testPlans);
```

Which I like.

Reply:
@mattpocock You can have that API with `testPlans(plans?, options?)` as well

Reply:
Sure, you just can't pass options without passing plans, which seems unintuitive